hugo/data/you_draw_it_karlsruhe.json

```diff
@@ -6,7 +6,7 @@
     "y_unit": "t. T.",
```

Comment:
Datei wird automatisch vom Python-Skript generiert, deshalb hier bitte keine Änderungen.
Daten nur in die `karlsruhe.csv` schreiben, dann tauchen sie auch hier auf (oder werden eben bei Bedarf geschätzt, siehe andere Diskussion).